### PR TITLE
Issue #215 - Attestation Results

### LINKED ISSUE #215: 8.3 Attestation Results - authorization
````
he passport model) or a Relying Party (in the background-check
   model) to indicate that the _Attester should not be treated as
   authorized_ and may be in need of remediation.  In some cases, it may
   even indicate that the Evidence itself cannot be authenticated as
   being correct.
````

Is there a different way to say this?  I assume in Passport, we’re not relying on the Attester to report itself as untrustworthy?
I think it’s more that it doesn’t get a passport, so it has nothing to hand the dude at the immigration desk?  i.e. untrustworthiness is indicated by lack of a passport?

### PR DESCRIPTION
**Pull request by nedmsmith**
Reworded 2nd and 3rd paragraphs fix #215

### CODE REVIEW

File: draft-ietf-rats-architecture.md [hunk 1]
```diff
@@ -994,17 +994,18 @@ Attestation Results may carry a boolean value indicating compliance or non-compl
...
 should not be treated as authorized and may be in need of remediation.  In some cases,
 it may even indicate that the Evidence itself cannot be authenticated as being correct.
 
-An Attestation Result that indicates compliance can be used by a Relying Party to make
-authorization decisions based on the Relying Party's appraisal policy.  The simplest such
-policy might be to simply authorize any party supplying a compliant Attestation Result
-signed by a trusted Verifier.  A more complex policy might also entail comparing information
-provided in the result against Reference Values, or applying more complex logic
-on such information.
+By default, the Relying Party assumes the Attester does not comply with Verifier appraisals.
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:1002]: This seems really awkward.

```suggestion
By default, the Relying Party does not believe the Attester to be compliant. 
```

File: draft-ietf-rats-architecture.md [hunk 2]
```diff
@@ -994,17 +994,18 @@ Attestation Results may carry a boolean value indicating compliance or non-compl
...
 
-An Attestation Result that indicates compliance can be used by a Relying Party to make
-authorization decisions based on the Relying Party's appraisal policy.  The simplest such
-policy might be to simply authorize any party supplying a compliant Attestation Result
-signed by a trusted Verifier.  A more complex policy might also entail comparing information
-provided in the result against Reference Values, or applying more complex logic
-on such information.
+By default, the Relying Party assumes the Attester does not comply with Verifier appraisals. 
+Upon receipt of an authentic Attestation Result and given the Appraisal Policy for Relying Party is satisfied. 
+The Attester is allowed to perform the prescribed actions or access.
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:1004]:
```suggestion
Upon receipt of an authentic Attestation Result and given the Appraisal Policy for Relying Party is satisfied, then
the Attester is allowed to perform the prescribed actions or access. 
```

**Review by mcr — CHANGES_REQUESTED** [review]
some clarifications needed

File: draft-ietf-rats-architecture.md (hunk 1 above)
Comment (dthaler) [reply draft-ietf-rats-architecture.md:1002]: +1 to mcr's suggested text

File: draft-ietf-rats-architecture.md (hunk 2 above)
Comment (dthaler) [reply draft-ietf-rats-architecture.md:1004]: Better, but "Appraisal Policy for Relying Party" also needs to be "Appraisal Policy for Attestation Results" which is the defined term.

**Review by dthaler — CHANGES_REQUESTED** [review]

File: draft-ietf-rats-architecture.md (hunk 2 above)
Comment (William-PanWei) [reply draft-ietf-rats-architecture.md:1004]: One more question: Checking the Attestation Result's authenticity is included in the Appraisal Policy for Attestation Results or not?
I.e., should we call an Attestation Result authentic only after the Appraisal Policy for Attestation Results is satisfied?

**Review by William-PanWei — CHANGES_REQUESTED** [review]

File: draft-ietf-rats-architecture.md (hunk 2 above)
Comment (mcr) [reply draft-ietf-rats-architecture.md:1004]: okay, what wording change do you suggest?

**Pull request merged**